cli/src/cli_util.rs

```diff
@@ -821,16 +821,25 @@ impl WorkspaceCommandHelper {
             std::env::var("HOME").map(|x| Path::new(&x).join(".config"))
         }
 
+        let resolve_excludes_file_path = |config: &gix::config::File| {
+            let path = get_excludes_file_path(config)?;
+            // The configured path is usually absolute, but if it's relative,
+            // the "git" command would read the file at the work-tree directory.
+            Some(self.workspace_root().join(path))
+        };
```

Comment:
Why not bake this into `get_excludes_file_path()`? And should it only be done in the `core.excludesFile` case? Maybe it doesn't matter, assuming `xdg_config_home()` is always absolute.

Reply:
I assumed `get_excludes_file_path()` could be replaced with some gix function, but apparently not. Inlined it.